rust/crd/src/lib.rs

```diff
@@ -90,8 +90,10 @@ pub struct HdfsClusterSpec {
 #[derive(Clone, Debug, Deserialize, Eq, Hash, JsonSchema, PartialEq, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct HdfsClusterConfig {
+    // FIXME: This attribute does not seem to be read anywhere.
```

Comment:
Can we remove it then :) ? We auto format in the init containers if not already happened anyways?

Reply:
I'm happy to remove it, but would prefer to not bloat this PR any further with a breaking change.
Added it to CRD changes list